server/scripts/dbSeeding.js

```diff
@@ -55,8 +55,8 @@ const mainCitiesGPS = [Toronto, QuebecCity, Montreal, Calgary];
       }
 
       const mock = {
-        firstName: name.firstName(),
-        lastName: name.lastName(),
+        firstName: name.firstName().replace(/[0-9]|'/g, ""),
+        lastName: name.lastName().replace(/[0-9]|'/g, ""),
```

Comment:
@dproc96 these two line

Reply:
The regex will look for any number and ' and replace with an empty string

Reply:
Why are we removing apostrophes? Shouldn't we allow users to have an apostrophe in their name?

Reply:
I see what you mean. Let me look into it.

Reply:
Hey what do you think of the solution?